Src/PerformanceCollector/Shared/PerformanceCollectorModule.cs

```diff
@@ -447,12 +448,13 @@ private void ProcessCustomCounters()
             // assign and sanitize reportsAs
             int i = 0;
             foreach (PerformanceCounterCollectionRequest req in this.Counters)
-                    {
-                        // TODO remove workaround
-                        req.ReportAs = string.IsNullOrWhiteSpace(req.ReportAs)
-                                           ? req.PerformanceCounter.Trim('\\').Replace(@"\", @" - ")
-                                           : req.ReportAs;
-                        req.ReportAs = this.SanitizeReportAs(req.ReportAs, req.PerformanceCounter, ref i);
+            {
+                // TODO remove workaround
```

Comment:
There was an incorrect indentation, this is how github shows the difference after fixing it.